apps/api/src/scraper/scrapeURL/lib/extractDataAttributes.ts

```diff
@@ -0,0 +1,72 @@
+import { load } from "cheerio";
```

Comment:
Logic should be moved to `html-transformer` Rust shared library if you're up for it, since cheerio can freeze the event loop.